torchgeo/datasets/geo.py

```diff
@@ -396,7 +396,9 @@ def __init__(
                     i += 1
 
         if i == 0:
-            msg = f"No {self.__class__.__name__} data was found in `root='{self.root}'`"
+            msg = (
+                f"No {self.__class__.__name__} data was found in `root='{self.paths}'`"
```

Comment:
```suggestion
                f"No {self.__class__.__name__} data was found in `paths={self.paths!r}`"
```
This will look better if paths is a list

Reply:
Fixed